fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1202,6 +1267,17 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 	}
 }
 
+void updateCommitCostEstimation(RatekeeperData* self, UIDTransactionTagMap<TransactionCommitCostEstimation> const& costEstimation) {
+	for(auto it = self->storageQueueInfo.begin(); it != self->storageQueueInfo.end(); ++ it) {
+		auto tagCostIt = costEstimation.find(it->key);
+		if(tagCostIt == costEstimation.end()) continue;
+		for(const auto& [tagName, cost] : tagCostIt->second) {
+			it->value.tagCostEst[tagName] += cost;
+			it->value.totalWriteCosts += cost.getCostSum();
+			it->value.totalWriteOps += cost.getOpsSum();
+		}
+	}
+}
```

Comment:
Style nit - I see in various places that there are no newlines between functions and other substantive blocks. Would you mind adding a newline in these places?